zwift.sh

```diff
@@ -406,6 +402,6 @@ if [ $RC -ne 0 ]; then
 fi
 
 # Allow container to connect to X, has to be set for different UID
-if [ -n "$CONTAINER" ] && [ -x "$(command -v xhost)" ] && [ -z $WINE_EXPERIMENTAL_WAYLAND ]; then
-    xhost +local:$($CONTAINER_TOOL inspect --format='{{ .Config.Hostname  }}' $CONTAINER)
+if [ -n "$CONTAINER" ] && [ -x "$(command -v xhost)" ] && [ -z "$WINE_EXPERIMENTAL_WAYLAND" ]; then
+    xhost +local:"$($CONTAINER_TOOL inspect --format='{{ .Config.Hostname  }}' "$CONTAINER")"
```

Comment:
this inner quote of the "$CONTAINER" surely doesn't work?
```suggestion
    xhost +local:"$($CONTAINER_TOOL inspect --format='{{ .Config.Hostname }}' $CONTAINER)"
```